core/sbt-exercise/src/main/scala/com/fortysevendeg/exercises/sbtexercise/ExerciseCompilerPlugin.scala

```diff
@@ -176,7 +185,15 @@ object ExerciseCompilerPlugin extends AutoPlugin {
       result ← libraries.map(invokeCompiler(compiler, _)).sequenceU
```

Comment:
If it's a matter of traversing over sources and invoking the compiler and accumulating errors we could also have used `Validated` here. I'm not saying there is something wrong with this but since each compilation computation does not depend on the previous one an `Applicative` instance may be more appropriate. `Validated` comes for free with `NonEmptyList` semantics and support for the Applicative Builder syntax.